tests/testthat/test-forecaster-utils.R

```diff
@@ -1,3 +1,5 @@
+for (file in list.files(here::here("R"), recursive = TRUE, full.names = TRUE)) source(file)
```

Comment:
might want to replace this with a `source` to a file `R/load_all` which runs this, especially since I think we're losing `devtools::load_all` by making this no longer a package

Reply:
true, can do